### DISCUSSION TITLE
Speech Separation for unknown number of Mutiple speakers

### DISCUSSION THREAD

**Discussion by Yuva-I** [speechbrain/speechbrain#2643] [Feature Request]
Hi,

Is there any method to apply speech separation to an unknown number of speakers in an audio file instead of manually monitoring several speakers in an audio file and selecting the model?
and I have seen limitations of speech separation for 2 or 3-speaker mix audios, is there any way to bypass these limitations?

**Comment by Lean-ENPLAS** [comment]
hi, have you found any solutions to this?
